core/templates/dev/head/services/classifier/CountVectorizerService.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
```

Comment:
Maybe put these in templates/dev/head/domain/classifier. They're not specific to any one interaction, right?